chapters/annotations.tex

```diff
@@ -1863,34 +1880,76 @@ \subsection{Version Numbering}\label{version-numbering}
 The main release versions are ordered using the hierarchical numerical names, and follow the corresponding pre-release versions.
 The pre-release versions of the same main release version are internally ordered alphabetically.
 
+
 \subsection{Version Handling}\label{version-handling}
 
-In a top-level class, the version number and the dependency to earlier versions of this class are defined using one or more of the following annotations:
-% TODO: Syntax below is a mess: neither Modelica, pseudo-code record, nor grammar.
-\begin{itemize}
-\item
-  \lstinline!version = CURRENT-VERSION-NUMBER!\annotationindex{version}\\
-  Defines the version number of the model or package.
-  All classes within this top-level class have this version number.
-\item
-  \lstinline!conversion(noneFromVersion = VERSION-NUMBER)!\annotationindex{conversion}\\
-  Defines that user models using the \lstinline!VERSION-NUMBER! can be upgraded to the \lstinline!CURRENT-VERSION-NUMBER! of the current class without any changes.
-\item
-  \lstinline!conversion(from(version = Versions, [to=VERSION-NUMBER,] Convert))!\\
-  where \emph{Versions} is \lstinline!VERSION-NUMBER! \textbar{} \lstinline!{VERSION-NUMBER, VERSION-NUMBER, $\ldots$}! and \lstinline!Convert! is \lstinline!script="$\ldots$"! \textbar{} \lstinline!change={conversionRule(), $\ldots$, conversionRule()}!\\*[.5ex]
-  Defines that user models using the \lstinline!VERSION-NUMBER! or any of the given \lstinline!VERSION-NUMBER! can be upgraded to the given \lstinline!VERSION-NUMBER! (if the to-tag is missing this is the \lstinline!CURRENT-VERSION-NUMBER!) of the current class by applying the given conversion rules.
-  The script consists of an unordered sequence of \lstinline!conversionRule();! (and optionally Modelica comments).
-  The \lstinline!conversionRule! functions are defined in \cref{conversion-rules}.
-
-  \begin{nonnormative}
-  The to-tag is added for clarity and optionally allows a tool to convert in multiple steps.
-  \end{nonnormative}
-\item
-  \lstinline!uses(IDENT (version = VERSION-NUMBER [, versionBuild=INTEGER] [, dateModified=STRING] ) )!\annotationindex{uses}\\
-  Defines that classes within this top-level class use version \lstinline!VERSION-NUMBER! of classes within the top-level class \lstinline!IDENT!.
-\end{itemize}
+In a top-level class, the version number and the dependency to earlier versions of this class are defined using one or more of the annotations described in this section.
+
+\begin{annotationdefinition}[version]
+\begin{synopsis}\begin{lstlisting}
+/*literal*/ constant String version
```

Comment:
```suggestion
/*literal*/ constant String version;
```
I thought the original idea was to have a full declaration with semi-colon.
I noticed that one of the recent updates also skipped that.